articles/key-design-of-hdfs-architecture/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: engineering-education
+status: 
+published: 
+url: /engineering-education/key-design-of-hdfs-architecture/
+title: Key Design of HDFS architecture
+description: This article will describe a brief introduction to Big data Framework, the general Design of HDFS architecture, ways of configuring HDFS Data Storage policies, Colocation and it's benefits in HDFS.
+author: ruth-mare
+date: 2021-03-19T00:00:00-14:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/key-design-of-hdfs-architecture/hero.jpg
+    alt: HDFS Architecture cover image 
+---
+HDFS (Hadoop Distributed File System) is a big data distributed file system storage by Apache that is implemented within the hadoop framework and that needs to have several features of design implemented to work effectively in processing, distibuting and storing big data.
```

Comment:
```suggestion
HDFS (Hadoop Distributed File System) is a big data distributed file system storage by Apache. It is implemented within the Hadoop framework and that needs to have several features of design implemented to work effectively in processing, distributing, and storing big data.
```

Reply:
Please make use of [hemingwayapp](hemingwayapp.com) to write shorter and meaningful sentences.